### ISSUE TITLE
Squawk schema failed to parse Postgres response

### ISSUE THREAD

**Issue by sk-** [sbdchd/squawk#171]
With the following files I'm getting a `failed to parse Postgres response` error.

```sql
ALTER TABLE "table" ALTER CONSTRAINT "constraint" DEFERRABLE INITIALLY IMMEDIATE;
```

```
Problem linting SQL files: Squawk schema failed to parse Postgres response. This indicates a bug with Squawk. Please report this error to https://github.com/sbdchd/squawk. Schema error: missing field `location` at line 1 column 266
```

```sql
ALTER TABLE "table" ALTER CONSTRAINT "constraint" NOT DEFERRABLE;
```

```
Problem linting SQL files: Squawk schema failed to parse Postgres response. This indicates a bug with Squawk. Please report this error to https://github.com/sbdchd/squawk. Schema error: missing field `location` at line 1 column 246
```

#Version
```
$ squawk --version
squawk 0.8.1
```

**Comment by chdsbd** [comment]
This issue has been fixed in 0.8.2, which can be installed from NPM.
